quaint/Cargo.toml

```diff
@@ -115,12 +115,12 @@ optional = true
 branch = "vendored-openssl"
 
 [dependencies.rusqlite]
-version = "0.25"
+version = "0.27"
 features = ["chrono", "bundled", "column_decltype"]
 optional = true
 
 [dependencies.libsqlite3-sys]
-version = "0.22"
+version = "0.24"
```

Comment:
Let's maybe remove it from our dependencies and use `rusqlite::sys` instead so we are guaranteed to use the same version and they won't diverge.

Reply:
ah, actually cargo won't let them diverge since it links to a native library, it would be an error